main.go

```diff
@@ -401,7 +401,7 @@ func prepareConfig(parseOpts *cliOptions, conf *ffuf.Config) error {
 		}
 	}
 
-	// Auto-calibration strings
+        // Auto-calibration strings
```

Comment:
This whitespace change should be reverted.

Reply:
reverted